src/CampaignAdmin.php

```diff
@@ -193,7 +193,7 @@ protected function getListResource()
     protected function getChangeSetResource(ChangeSet $changeSet)
     {
         $hal = [
-            '_links' => [
```

Comment:
Hmm yeah ok...
The code is getting quite confusing switching between a reference link, the ChangeSetItemID and ObjectID...
@phalkunz are we able to standardise this to use either ChangeSetItemID or ObjectID+ObjectClass, and not use a mix of these?
Sorry for the mix up, it was quite confusing to read through the code review once @tractorcow pointed out the HAL specs...

Perhaps ChangeSetItemID would be the best, because it's always a unique identifier.